src/style.css

```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url("fonts/Avenir.ttc");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+  box-sizing: border-box;
```

Comment:
You set `box-sizing` parameter several times, so it can be set globally via `*`.